internal/release/services/release/release.go

```diff
@@ -282,6 +293,111 @@ func GetReleaseSchema(table string) *dynamodb.CreateTableInput {
 	}
 }
 
+// Function to send notifications to webhook
+func NotifyWebhook(payload Release) error {
```

Comment:
We should have a unit test for this to confirm that the webhook is being called and pushing valid JSON data.
Take a look at https://pkg.go.dev/net/http/httptest#Server - you're going to need to handle the fact that the handler is running in a different thread than the test so I suggest using channels to sync the POST and test together.

psuedo-code
```
        resultsChan := make(chan bool)
        ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		// validate request
                resultsChan <- true/false depending on success failure
	}))
	defer ts.Close()
        result := <- resultsChan
       
```